app/src/androidTest/java/com/android/sample/createRecipe/IngredientSearchScreenTest.kt

```diff
@@ -120,7 +119,6 @@ class IngredientSearchScreenTest {
 
     composeTestRule.onNodeWithTag(CONFIRMATION_BUTTON, useUnmergedTree = true).performClick()
     verify(mockNavigationActions).navigateTo(Screen.CREATE_RECIPE_LIST_INGREDIENTS)
-    assertEquals(ingredientViewModel.ingredientList.value, listOf(mockIngredients[0]))
```

Comment:
Is there a reason for why this verification was removed ?

Reply:
Since the assertion no longer holds true after the modification, I’ve updated it with a new check.